test/test_helper.ts

```diff
@@ -361,18 +383,37 @@ class TestHelper {
         };
     };
 
-    fakeUser = () => {
+    fakeUser = (): UserProfile => {
         return {
             email: this.fakeEmail(),
-            allow_marketing: true,
-            password: PASSWORD,
```

Comment:
Why the removal?

Reply:
They are not defined in the typescript type, so in theory they are not used in the code.